src/admin/storage-info/index.ts

```diff
@@ -0,0 +1,59 @@
+import { grpc } from "@improbable-eng/grpc-web"
+import {
+  ListStorageInfoRequest,
+  ListStorageInfoResponse,
+  StorageInfoRequest,
+  StorageInfoResponse,
+} from "@textile/grpc-powergate-client/dist/powergate/admin/v1/admin_pb"
+import { AdminServiceClient } from "@textile/grpc-powergate-client/dist/powergate/admin/v1/admin_pb_service"
+import { Config } from "../../types"
+import { promise } from "../../util"
+
+export interface StorageInfo {
+  /**
+   * Get the current storage state of a cid.
+   * @param userId The user id to query.
+   * @param cid The cid to query.
+   * @returns The current storage state of the cid.
+   */
+  get: (userId: string, cid: string) => Promise<StorageInfoResponse.AsObject>
+
+  /**
+   * Lists the current storage state for many or all user ids and cids.
+   * @param cids Optional list of cids to filter the results by.
+   * @returns An object containing a list of storage info.
+   */
+  list: (userIds?: string[], cids?: string[]) => Promise<ListStorageInfoResponse.AsObject>
+}
```

Comment:
New API for getting `StorageInfo` data which describes the current storage state of data in powergate (data that is already/currenlty actually stored, not processing, queued, etc)